Modules/ProfileManager/ProfileManager.lua

```diff
@@ -235,6 +253,99 @@ function ProfileManager:AddTarget()
     end
 end
 
+function ProfileManager:PruneBelowLevel(minLevel, nop)
```

Comment:
This function doesn't seem to fully use LUA scoping. Instead of passing in the `log` via a function parameter and then renaming it `_log`, just use it directly.